lib/index.js

```diff
@@ -54,6 +59,15 @@ function downloadFromInfoCallback(stream, info, options) {
   req.on('response', function(res) {
     stream.emit('response', res);
     if (res.statusCode !== 200) {
+      if(res.statusCode == 302) {
+        //Redirection header//
```

Comment:
The current comment style right now is space after `//`, no additional `//` at the end, complete sentences with period at end.